### ISSUE TITLE
Worth adding some subset of stdlib Logging?

### ISSUE THREAD

**Issue by jgoldfar** [JuliaLang/Compat.jl#536]
A more refined version of (e.g.)
```julia
macro info(s)
    :(info($s))
end
```
would make it easier to write code that works on v0.6 and v0.7/1.0, or would the recommendation be to import MicroLogging or some other framework on v0.6?

**Comment by fredrikekre** [comment]
> I never think to check for unexported functionality from Compat...

All implemented functionality should be listed in Compats README.

**Comment by jgoldfar** [comment]
Yes, it is there. I don’t make a habit of reading the README file looking for unexported functionality either. Will do next time. Thanks!
